README.md

```diff
@@ -1 +1,147 @@
-# Hello-World
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+HOW TO USE:
+This is a simple HTML template.
+- Table of Contents
+- About the Project
+  - Built With: HTML, CSS
+- Getting Started
+- Authors : Abenezer 
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements : To Microverse
+- License MIT.
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./microverse-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Hello world project</b></h3>
+
+</div>
```

Comment:
- [ OPTIONAL ] Please remove this content because I don't think you need it and that would make your readme file clean.  You can remove the Microverse logo as well. It is there like a placeholder for you in case you have your own logo to put there. 😉 😊